### PR TITLE
Only run Debug when actually running debug

### PR DESCRIPTION
**Pull request by dorana**
- Set so that debug is passed down correctly
- Cleaned up how we checkForceRelease
- Removed Output when running release mode
- Included idea (Rider project files) in gitignore

### CODE REVIEW

File: RazorEnhanced/RazorEnhanced/Scripts.cs [hunk 1]
```diff
@@ -306,7 +306,7 @@ private void AsyncStart()
                     if (ext.Equals(".cs", StringComparison.InvariantCultureIgnoreCase))
                     {
                         CSharpEngine csharpEngine = CSharpEngine.Instance;
-                        bool compileErrors = csharpEngine.CompileFromFile(fullpath, true, out List<string> compileMessages, out Assembly assembly);
+                        bool compileErrors = csharpEngine.CompileFromFile(fullpath, false, out List<string> compileMessages, out Assembly assembly);
```
Comment (caporalesimone) [inline RazorEnhanced/RazorEnhanced/Scripts.cs:309]: Do you expect to run scripts always in release?
why don't use debug variable to choose which version compile?

Comment (dorana) [reply RazorEnhanced/RazorEnhanced/Scripts.cs:309]: When running from script tab or hotkey yes. The debug flag will be passed if run in debug from the code editor

Comment (dorana) [reply RazorEnhanced/RazorEnhanced/Scripts.cs:309]: Changed from true to debug in script.cs

Comment (caporalesimone) [reply RazorEnhanced/RazorEnhanced/Scripts.cs:309]: I attach visual studio to re and run script from the tab. In this case is useful execute it in debug also from the tab

Comment (dorana) [reply RazorEnhanced/RazorEnhanced/Scripts.cs:309]: I can add a checkbox that will be used to toggle debug then, simple thing to add

Comment (caporalesimone) [reply RazorEnhanced/RazorEnhanced/Scripts.cs:309]: Agree with this idea

Comment (dorana) [reply RazorEnhanced/RazorEnhanced/Scripts.cs:309]: Also, should the PR remain on 0.9 or would you prefer it on the 0.8 branch?

Comment (caporalesimone) [reply RazorEnhanced/RazorEnhanced/Scripts.cs:309]: Let's ask @credzba  what he thinks.
The change is quite small.
Visually is only a checkbox in the script tab and those small fixes.
I would like to see it in 0.8.

Comment (cesare-montresor) [reply RazorEnhanced/RazorEnhanced/Scripts.cs:309]: I would say, push it on both

Comment (credzba) [reply RazorEnhanced/RazorEnhanced/Scripts.cs:309]: It is fine by me,  I'll cherry-pick it over to 0.8 if its not there yet, I am a bit behind

**Pull request merged**